inst/tests/tests.Rraw

```diff
@@ -18286,3 +18286,17 @@ test(2246.1, DT[, data.table::shift(b), by=a], DT[, shift(b), by=a], output="GFo
 test(2246.2, DT[, data.table::first(b), by=a], DT[, first(b), by=a], output="GForce TRUE")
 test(2246.3, DT[, data.table::last(b), by=a], DT[, last(b), by=a], output="GForce TRUE")
 options(old)
+
+# unique.data.table doesn't return a mutable alias, #5960
+# unique nrows > 1 should return different address object (even before this fix)
+DT1 = data.table(a = 1:2, b = 3:4)
+DT2 = unique(DT1)
+test(2247.1, address(DT1) != address(DT2), TRUE)
+# unique nrows == 1 should return different address object (after this fix)
+DT1 = data.table(a = 1, b = 3)
+DT2 = unique(DT1)
+test(2247.2, address(DT1) != address(DT2), TRUE)
```

Comment:
`test` has the signature `test(num, x, y=TRUE)` so we could always save the TRUE at the end